src/pageComponents/joinUsPage/JoinUsPage.tsx

```diff
@@ -21,9 +21,9 @@ export const JoinUsPageContent = ({ content }: { content: JoinUsSanityContent })
   const { language } = useLanguage()
 
   return (
-    <Stack sx={{ justifyContent: 'center', alignItems: 'center', width: '100%' }}>
-      <Stack sx={{ textAlign: 'center', alignItems: 'center', width: '100%', maxWidth: DEFAULT_LAYOUT_MAXWIDTH, justifyContent: 'center' }}>
-        <Typography variant='h1' fontWeight={700} width={'100%'}>
+    <>
+      <ContentWrapper>
```

Comment:
Alle andre steder har du puttet den i page, så ville holdt meg til det.

Ser at du har fragmentert litt her for siste seksjon, men skal være helt løstbart i en styling